Index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>Edward's Portfolio</title>
+</head>
+<body>
+    <main>
+        <nav class="navbar">
+        <div class="nav-items">
+         <a href="" class="logo">My Logo</a>
+         <a href=""><i class="fas fa-bars"></i></a>
+        
+         <!--Hamburger menu-->
+        <div class="hamburger-menu">
+           <div class="line line-1"></div>
+           <div class="line line-2"></div>
+           <div class="line line-3"></div>
+        </div>
+        </div> 
+        </nav>
+        
+
+    <section class="container">
+
+        <div class="intro">
+            <h1>I'm Edward</h1> 
+            <h1>Glad to see you!</h1>
+           <p>
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+           </p>
+        </div>
+  
+      
+        <div class="contact"> 
+            <a href="#" class="connect">LET'S CONNECT</a>
+        </div> 
+        <div class="social">
+            <ul class="media">
+
+              <li><a href="" class="twitter"><img src="./twitter.png" width="15" height="18" alt="twitter" ></a></li> 
+              <li><a href="#" class="linkedin" ><img src="./linkedin.png" width="15" height="20" alt="linkedin"></a></li>
+              <li><a href="#" class="medium"><img src="./medium.png" width="15" height="18" alt="medium"></a></li>
+              <li><a href="#" class="github"><img src="./github.png" width="15" height="18" alt="github"></a></li> 
+              <li><a href="#" class="vector"><img src="./Vector.png" width="15" height="18" alt="vector"></a></li> 
+            </ul>
+        </div>
```

Comment:
- [ ] **[REQUIRED CHANGES]** You have done a great effort to add all of the images and the image background! But I've noticed that most of your images are broken. Please be sure to change the images. Did you change the image extensions (i.e from `.jpg` to `.png`)?

- [ ] **[REQUIRED CHANGES]** Remember to remove the margins from the `toolbar` and the `section`. As proposed by the figma template, they have to be added as padding (vertically 114px and horizontally 24px) on the section.